### PR TITLE
Registration Page: Mentor Registration Page

### LINKED ISSUE #81: Frontend: Registration Page
When I am registering and login in for the first time, I want to choose the name and the class or company that I am part of, depending if I am student, a mentor or admin, or skills if I am a mentor, so I can use the web app accordingly to my role.

Create a page below that considers the data retrieved with the endpoint developed on the issue to show the list #58 and the page component created on this issue #61 - RegistrationPage - src/client/containers/RegistrationPage/RegistrationPage.component.js

If student the app should show this:
image

If mentor the app should show this:
image

If admin the app should show this:
image

IMPORTANT: make sure you use existing components, if needed change the existing components or work together with the developer that created those same components.

### PR DESCRIPTION
**Pull request by Deepti-Dk**
# Description

Mentor registration page created using existing components.

Fixes #81 (Part 3 of 3)

# How to test?

Tested through storybook and is working and responsive.

# Checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Review by bytewiz — COMMENTED** [review]
Seems good to me, are you supposed to delete the `Registration page` route?

**Comment by Deepti-Dk** [comment]
> Seems good to me, are you supposed to delete the `Registration page` route?

Yes, as 3 separate

> Seems good to me, are you supposed to delete the `Registration page` route?

Yes, as 3 separate registration pages are now created for mentor, admin and student (initially, the plan was to create just one page for all roles but later we decided to do 3 separate pages) and just updated route name, as well to be more consistent with other pages.

**Review by tpetrina — COMMENTED** [review]
Fix merge conflict - have some changes already in another PR.

**Review by tpetrina — DISMISSED** [review]

**Review by nuha90 — DISMISSED** [review]

**Review by nuha90 — DISMISSED** [review]

File: src/client/App.js [hunk 1]
```diff
@@ -43,8 +42,8 @@ function App() {
             <Route exact path="/selection">
               <SelectionPage />
             </Route>
-            <Route exact path="/registration/student">
-              <StudentRegistrationPage />
+            <Route exact path="/registration/mentor">
+              <MentorRegistrationPage />
```
Comment (tpetrina) [inline src/client/App.js:46]: Same as in the other PR, you removed student registration.

**Review by tpetrina — CHANGES_REQUESTED** [review]

File: src/client/App.js (hunk 1 above)
Comment (Deepti-Dk) [reply src/client/App.js:47]: updated

**Review by tpetrina — APPROVED** [review]

**Review by MaherHussain — APPROVED** [review]
go ahead

**Review by tpetrina — APPROVED** [review]

**Review by MaherHussain — APPROVED** [review]

**Pull request merged**